tests/test_diffeq/test_odesolver.py

```diff
@@ -2,10 +2,24 @@
 
 import numpy as np
 
-from probnum.diffeq import ConstantSteps, ODESolver, logistic
+from probnum.diffeq import ConstantSteps, ODESolution, ODESolver, logistic
 from probnum.random_variables import Constant
 
 
+class MockODESolution(ODESolution):
+    def __init__(self, t, y):
+        self._t = t
+        self._y = y
+
+    @property
+    def t(self):
+        return self._t
+
+    @property
+    def y(self):
+        return self._y
+
+
```

Comment:
this is the minimum implementation of ODE solutions. One could also provide this in the source code, but since we do not have solvers that need this (currently), I find it unnecessary.